src/lib/components/ui/Card/Card.jsx

```diff
@@ -11,15 +11,31 @@ export const Card = (props) => {
     inList,
     raised,
     type,
+    variant,
   } = props;
 
-  let typeClass = styles.isTypeFlat;
-  if (type === 'warning') {
+  const autoDetectedVariant = type === null ? variant : 'bordered';
```

Comment:
Maybe the name is confusing. What I want to achieve is to force-set `variant` to `bordered` if a `type` is specified. In other words, one prop can override another prop. It may look weird in code but it makes the use of the component easier.

Reply:
Yeah, name is confusing and I know now purpose of it. But I would be pleased if we can change it to better name because other people would have meet the same confusion as I did. Maybe `overrideVariant` or something like this?

Reply:
OK, I'll change it to `overriddenVariant`.